x-pack/plugins/security_solution/public/resolver/view/panel.test.tsx

```diff
@@ -160,23 +189,39 @@ describe(`Resolver: when analyzing a tree with no ancestors and two children and
 
   it('should have 3 nodes (with icons) in the node list', async () => {
     await expect(
-      simulator().map(() => simulator().testSubject('resolver:node-list:node-link:title').length)
-    ).toYieldEqualTo(3);
-    await expect(
```

Comment:
This test asserted that there were 3 titles, then at a later point in time, 3 icons. The new test asserts that 3 icons and 3 titles coexist.